src/plugins/dragresize_ie11.js

```diff
@@ -986,7 +986,11 @@
 
 			const image = wrapper.$.querySelector('img');
 
-			image.removeAttribute('style');
+			let imageStyles = image.getAttribute('style');
+			let widthStyles = /(width:.*?;)/g.exec(imageStyles);
+			let widthStyle = widthStyles[0];
```

Comment:
Seems all of these `let` should be `const`, right?

Also, about that regex: `.*?` is a non-greedy match (good), but `*` implies that zero-width is acceptable and it isn't, is it? ie. `.+?` would be a more accurate specification of what we're looking for. Not that I think we'll get a zero-width value there in practice, just that I think in general it's best to keep regexes as "tight" as possible.

Reply:
Yeah, they can all be `const`.  Wasn't aware that's the convention but that makes sense.

And right, `.+?` is definitely better here